static/dswc_leaderboard.js

```diff
@@ -10,74 +9,93 @@ var dataSource = "";
+        /* {'key': 'boatNum', class: 'text-center'}, */
+
+        { key: 'points', class: 'text-center' }
     ];
-    for (var b in data) {
+    for (var result of data) {
         var dRec = {};
-        dRec.crew = data[b].crew.join(', ');
-        /* dRec.boatNum = data[b].boatNum; */
-        dRec.points = data[b].points;
-        dRec.place = data[b].place;
-        dRec._cellVariants = {};
-        for (var r in data[b].races) {
-            var raceStr = parseInt(r, 10) + 1;
-            raceStr = 'R' + raceStr;
+
+        /* dRec.boatNum = result.boatNum; */
+        dRec.place = result.place;
+        dRec.points = result.points;
+        dRec.crew =
+            (result.place === 1
+                ? '🥇'
+                : result.place === 2
+                ? '🥈'
+                : result.place === 3
+                ? '🥉'
+                : '') + result.crew.join(', ');
+
+        // First, second and third positions
+        var highlightResult =
+            result.place === 1
+                ? 'primary'
+                : result.place === 2 || result.place === 3
+                ? 'success'
+                : '';
+        dRec._cellVariants = { place: highlightResult };
+
+        for (var r in result.races) {
+            var raceStr = 'R' + parseInt(r, 10) + 1;
             var fRec = {
                 key: raceStr,
                 class: 'text-center'
             };
             fields.push(fRec);
-            if (data[b].races[r].discard) {
-                dRec[raceStr] = '(' + data[b].races[r].place + ')';
+
+            dRec[raceStr] = result.races[r].place;
+
+            if (result.races[r].discard) {
+                dRec[raceStr] = '(' + dRec[raceStr] + ')';
+                dRec._cellVariants[raceStr] = 'danger';
+            } else if (result.races[r].flag) {
+                dRec._cellVariants[raceStr] = 'warning';
             } else {
-                dRec[raceStr] = data[b].races[r].place;
-            }
-            if (data[b].races[r].flag) {
-                dRec._cellVariants[raceStr] = 'success';
+                dRec._cellVariants[raceStr] = 'info';
             }
         }
         dData.push(dRec);
     }
-   obj.summaryData.items = dData;
-   obj.summaryData.fields = fields;
+    obj.summaryData.items = dData;
+    obj.summaryData.fields = fields;
 }
 
 var allSeries = new Vue({
     el: '#allSeries',
     data: {
         seriesList: []
     },
-    mounted () {
+    mounted() {
         axios
             .get(dataSource + '/api/v1.0/listSeries')
-            .then(response => (this.seriesList = response.data))
+            .then(response => (this.seriesList = response.data));
     }
 });
 
 Vue.component('result-table', {
     props: ['title', 'tabledata'],
     template: `
         <div class='result-table'>
-            <h3>{{ title }}</h3>
+            <h3>Results for <span class="text-primary">{{ title.split('_')[0] }} {{ title.split('_')[1] }}</span></h3>
```

Comment:
```suggestion
            <h3>Results for <span class="text-primary">{{ title.replace('_', ' ') }}</span></h3>
```

I wonder if we should add a css element to capitalise the first letter of the title, can we use css to extend the text-primary class?:
```
span.resultsTitle {
  text-transform: capitalize;
  ...
}
```